uint32 slave
uint16 statusWord
int32 actualPosition
int32 actualVelocity
int8 opMode
